### PR TITLE
Added emitter to the collision exceptions on _Ready

### LINKED ISSUE #1101: Change AgentProjectile to use collision exceptions
AgentProjectile could use the collision exceptions like engulfing on the Emitter to also be able to pass through the emitter, instead of just ignoring it in the contact callback.

The microbe code is not yet in master: https://github.com/Revolutionary-Games/Thrive/pull/1100

### PR DESCRIPTION
**Pull request by CoreyHendrey**
I tested this and it seems to work.
It's for issue #1101.

I don't remove the Emitter from the Collision exceptions, as it appears it is already disposed at that point. I could however add the remove inside a try/catch block on Destroy if that seems like a better idea, and just catch the Dispose exception or something. Not sure if that is needed though.

fixes #1101

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
add to whitelist

File: src/microbe_stage/AgentProjectile.cs [hunk 1]
```diff
@@ -1,4 +1,5 @@
 using Godot;
+using System;
```
Comment (hhyyrylainen) [inline src/microbe_stage/AgentProjectile.cs:2]: CI is probably failing because you added this include.

Comment (CoreyHendrey) [reply src/microbe_stage/AgentProjectile.cs:2]: Ah will remove it.

**Comment by CoreyHendrey** [comment]
> This is pretty much the change I had in mind, but this is pretty difficult to test if this works correctly. @Kasterisk ?

I attempted to test it by removing all the checks on firing toxins, so that I could fire them like crazy over the screen - haha.

Pretty flawed test though.

**Pull request merged**